{
    "docs": "https://docs.ansible.com/ansible-core/2.15/dev_guide/testing/sanity/yamllint.html", 
    "results": [
        {
            "message": "The test `ansible-test sanity --test yamllint` [[explain](https://docs.ansible.com/ansible-core/2.15/dev_guide/testing/sanity/yamllint.html)] failed with 2 errors:", 
            "output": "plugins/modules/ovh/dns/ovh_dns_record.py:88:5: error: EXAMPLES: syntax error: expected <block end>, but found '<block mapping start>' (syntax)\nplugins/modules/ovh/dns/ovh_dns_record.py:88:5: unparsable-with-libyaml: while parsing a block mapping - did not find expected key"
        }
    ], 
    "verified": false
}
